tests/LSP1UniversalReceiver/LSP1UniversalReceiverDelegateUP.behaviour.ts

```diff
@@ -1239,18 +1261,845 @@ export const shouldBehaveLikeLSP1Delegate = (
+          "0xda1f85e400000000000000000000000000000000cafecafe",
+        ]);
+      });
+
+      it("should pass", async () => {
+        const balance = await token.balanceOf(
+          context.universalProfile1.address
+        );
+
+        const tokenTrasferCalldata = token.interface.encodeFunctionData(
+          "transfer",
+          [
+            context.universalProfile1.address,
+            context.accounts.owner1.address,
+            balance,
+            true,
+            "0x",
+          ]
+        );
+
+        expect(
+          await context.universalProfile1
+            .connect(context.accounts.owner1)
+            ["execute(uint256,address,uint256,bytes)"](
+              0,
+              token.address,
+              0,
+              tokenTrasferCalldata
+            )
+        ).to.not.be.reverted;
+      });
+
+      it("should emit UniversalReceiver event", async () => {
+        const balance = await token.balanceOf(
+          context.universalProfile1.address
+        );
+
+        const tokenTrasferCalldata = token.interface.encodeFunctionData(
+          "transfer",
+          [
+            context.universalProfile1.address,
+            context.accounts.owner1.address,
+            balance,
+            true,
+            "0x",
+          ]
+        );
+
+        const tokensSentBytes32Value = ethers.utils.hexZeroPad(
+          balance.toHexString(),
+          32
+        );
+
+        const tokenTransferData = (
+          context.universalProfile1.address +
+          context.accounts.owner1.address.substring(2) +
+          tokensSentBytes32Value.substring(2)
+        ).toLocaleLowerCase();
+
+        const lsp1ReturnedData = ethers.utils.defaultAbiCoder.encode(
+          ["bytes", "bytes"],
+          ["0x", "0x"]
+        );
+
+        await expect(
+          context.universalProfile1
+            .connect(context.accounts.owner1)
+            ["execute(uint256,address,uint256,bytes)"](
+              0,
+              token.address,
+              0,
+              tokenTrasferCalldata
+            )
+        )
+          .to.emit(context.universalProfile1, "UniversalReceiver")
+          .withArgs(
+            token.address,
+            0,
+            LSP1_TYPE_IDS.LSP7Tokens_SenderNotification,
+            tokenTransferData,
+            lsp1ReturnedData
+          );
+      });
+
+      it("should de-register the asset properly", async () => {
+        const balance = await token.balanceOf(
+          context.universalProfile1.address
+        );
+
+        const tokenTrasferCalldata = token.interface.encodeFunctionData(
+          "transfer",
+          [
+            context.universalProfile1.address,
+            context.accounts.owner1.address,
+            balance,
+            true,
+            "0x",
+          ]
+        );
+
+        await context.universalProfile1
+          .connect(context.accounts.owner1)
+          ["execute(uint256,address,uint256,bytes)"](
+            0,
```

Comment:
same as above, use `OPERATIONS_TYPE` here to improve readability of the test

```suggestion
            OPERATIONS_TYPE.CALL,
```